### ISSUE TITLE
DocBlock Issue 

### ISSUE THREAD

**Issue by ahmadawais** [wesbos/cobalt2#98]
I think this is a DocBlock issue that the 

``` php
@access private
```

Where `private` gets a color other than the comment color?

**Comment by wesbos** [comment]
Is there a scope for this? Does it work on other themes?

The theme shouldn't have to cater to every-single use case, its the syntax highlighter that does this.

**Comment by ahmadawais** [comment]
@wesbos  Yes there is a scope and this is part of the [standard documentation practices for WordPress](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/).

The core code is full of it. It is very disturbing with the wrong color.
See fo yourself ![](https://i.imgur.com/6L7eeGn.png) 

Here's the scope 
```
Scope:                         embedding.php
                               text.html.basic
                               meta.embedded.block.php
                               source.php
                               comment.block.documentation.phpdoc.php
                               storage.modifier.php
------
```
Guide me, I can create a pull request.

**Comment by ahmadawais** [comment]
Any luck?

**Comment by ahmadawais** [comment]
@wesbos  Here's another example

![](https://i.imgur.com/vQ1057P.png)

You can test the code at your end. https://gist.github.com/f8c0ca40009294316cbd
